test/unit/server.test.js

```diff
@@ -8,7 +8,29 @@
  *       Lewis Daly - lewisd@crosslaketech.com                            *
  **************************************************************************/
 'use strict'
+const { OracleDatabase } = require('../../src/db')
+const { createServer } = require('../../src/server')
 
+const fakeConfig = {
```

Comment:
I would use here real config, elsewhere there is a need to recheck this fakeConfig for every change in the real one....

Reply:
done